mulighetsrommet-api/src/test/kotlin/no/nav/mulighetsrommet/api/repositories/TiltaksgjennomforingRepositoryTest.kt

```diff
@@ -60,12 +61,41 @@ class TiltaksgjennomforingRepositoryTest : FunSpec({
                 status = Tiltaksgjennomforingsstatus.AVSLUTTET,
                 tilgjengelighet = Tilgjengelighetsstatus.Ledig,
                 antallPlasser = null,
-                avtaleId = gjennomforing1.avtaleId
+                avtaleId = gjennomforing1.avtaleId,
+                ansvarlige = emptyList(),
             )
 
             tiltaksgjennomforinger.delete(gjennomforing1.id)
 
-            tiltaksgjennomforinger.getAll(filter = AdminTiltaksgjennomforingFilter()).second shouldHaveSize 1
+            tiltaksgjennomforinger.getAll(filter = AdminTiltaksgjennomforingFilter()).shouldBeRight().second shouldHaveSize 1
+        }
+    }
+
+    context("TiltaksgjennomforingAnsvarlig") {
+        test("Ansvarlige crud") {
+            val tiltaksgjennomforinger = TiltaksgjennomforingRepository(database.db)
+
+            val ident1 = "N12343"
+            val ident2 = "Y12343"
+            val gjennomforing = gjennomforing1.copy(ansvarlige = listOf(ident1, ident2))
+
+            tiltaksgjennomforinger.upsert(gjennomforing).shouldBeRight()
+            val gjennomforingFraDB = tiltaksgjennomforinger.get(gjennomforing.id).shouldBeRight()!!
```

Comment:
Evt. kan du benytte `.should { ... }` (krever egen import) i kombinasjon med `.shouldBeRight()` for mer konsise assertions (kommer muligens an på øyet som ser):

```
tiltaksgjennomforinger.get(gjennomforing.id).shouldBeRight().should {
    it.ansvarlige.shouldContainExactlyInAnyOrder(ident1, ident2)
}
```

Reply:
Nice, skal teste det :+1: